netio.c

```diff
@@ -401,9 +401,9 @@ query_done(query_t query) {
 		const char *msg = or_else(query->saf_msg, "");
 
 		if (query->saf_cond == sc_limited)
-			fprintf(stderr, "Query limited: %s\n", msg);
+			fprintf(stderr, "Database limit: %s\n", msg);
 		else if (query->saf_cond == sc_failed)
-			fprintf(stderr, "Query failed: %s\n", msg);
+			fprintf(stderr, "Database failure: %s\n", msg);
```

Comment:
IMHO:
"Database failure" sounds like a bad fatal thing, like a database having its disks fail.
"Query failed" is not as alarming.

Reply:
will change to "Database result: %s", since the %s will describe the actual reason for failure ("limited"), all i'm trying to do is make it clear to the user that the error didn't occur locally.